MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -4752,8 +4752,8 @@ uint64_t mvkGetRegistryID(id<MTLDevice> mtlDevice) {
 bool mvkSupportsBCTextureCompression(id<MTLDevice> mtlDevice) {
 #if MVK_IOS || MVK_TVOS || MVK_MACCAT
 	return false;
-#endif
-#if MVK_MACOS && !MVK_MACCAT
+ #endif
+ #if MVK_MACOS && !MVK_MACCAT
```

Comment:
I assume these additional spaces are a typo? If so, can we remove them?

Reply:
Of course.